Improved Use, Emotes, and Sounds

SQUIDzillaNFT opened the pull request:
Emote when detecting so its not flopping around, auto stop when leaving area, Detecting sounds(interact-sound), and Key binds to start or stop Detecting. Also added better Notifications, fixed notification word errors. Added dependency Interact-sound with metaldetector.ogg and metaldetected.ogg Will provide those if needed.

SQUIDzillaNFT committed:
Improved Use, Emotes, and Sounds

Emote when detecting, auto stop when leaving area, Detecting sounds, and Key binds to start stop Mining. Also added better Notifications.

SQUIDzillaNFT reviewed (commented):
I was inspired by lation scripts detecting, and though it would be cool to make this free and alot more robust and fun for everyone.

Mustachedom commented:
Much appreciated! This has been on my back burner for a big update so thank you for cutting out a big chunk of the work :) if you want to push the sound files tbat would be amazing :)

SQUIDzillaNFT committed:
added sounds if not included in interact-sounds

place in interact-sound/client/html/sounds/

SQUIDzillaNFT commented:
Added sounds in a sound folder.

SQUIDzillaNFT committed:
Update Model Loading

Model sometimes wouldn't load on startup with "cant get entity ID" until script was restarted. Model will always load in now without Entity Issue.
